test/cookie.test.js

```diff
@@ -943,8 +864,79 @@ test('if cookies are not set, then the handler creates an empty req.cookies obje
   })
 })
 
+test('share options for setCookie and clearCookie', (t) => {
```

Comment:
Could you avoid moving the test?
it makes hard to review the PR

Reply:
Of course, I must've missed that change before committing.